### DISCUSSION TITLE
Error 500 when updating Teampass

### DISCUSSION THREAD

**Discussion by c1thunder** [nilsteampassnet/TeamPass#3999] [General]
I run git pull to update my teampass. When i tried to access https://<your_teampass_instance>/install 
/upgrade.php, its throwing me 500 error. Any idea?

![image](https://github.com/nilsteampassnet/TeamPass/assets/20919681/0a1a656b-be1a-4693-98b2-e568147ca548)

**Comment by nilsteampassnet** [comment]
@c1thunder 
on master branch?
Please check apache error log

**Reply by c1thunder** [reply]
The log is 

> [Tue Jan 09 15:59:39.472239 2024] [mpm_prefork:notice] [pid 879] AH00169: caught SIGTERM, shutting down
>[Tue Jan 09 15:59:57.321310 2024] [mpm_prefork:notice] [pid 861] AH00163: Apache/2.4.41 (Ubuntu) configured -- resuming normal operations
>[Tue Jan 09 15:59:57.330584 2024] [core:notice] [pid 861] AH00094: Command line: '/usr/sbin/apache2'

**Reply by nilsteampassnet** [reply]
@c1thunder
Ok nothing interesting.
And in your webbrowser console?
Error 500 is related to server, so there must be kind of error somewhere.
Is the folder 'install' existing?

**Comment by c1thunder** [comment]
@nilsteampassnet 
that was also the one i suspected at first, but the folders seem fine. The system running well before i updated, is my step wrong?
![image](https://github.com/nilsteampassnet/TeamPass/assets/20919681/d07d34a9-8d53-4cfa-ac7d-045ed54c405c)
here are the steps that i was using. I obfuscate my local user for privacy reasons
![image](https://github.com/nilsteampassnet/TeamPass/assets/20919681/2dd76dcc-241d-46f0-873b-e2972fbbefb4)
